### PR TITLE
[Tachyon-503]Fixed:loadufs creates a directory under tachyon if the underlying folder does not exist.

### PR DESCRIPTION
**Pull request by kvaishnav**
Throw a Filenotfound exception if the underfs folder is not available, before creating the directory under tachyon

### CODE REVIEW

File: bin/tachyon-workers.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ TACHYON_LIBEXEC_DIR=${TACHYON_LIBEXEC_DIR:-$DEFAULT_LIBEXEC_DIR}
 HOSTLIST=$TACHYON_CONF_DIR/workers
 
 for worker in `cat "$HOSTLIST" | sed  "s/#.*$//;/^$/d"`; do
-  echo -n "Connecting to $worker as $USER... "
+  echo "Connecting to $worker as $USER..."
```
Comment (calvinjia) [inline bin/tachyon-workers.sh]: Is this more readable without the `-n`?

File: clients/unshaded/src/main/java/tachyon/client/UfsUtils.java [hunk 2]
```diff
@@ -111,6 +112,10 @@ public static void loadUnderFs(TachyonFS tfs, TachyonURI tachyonPath, TachyonURI
     // create the under FS handler (e.g. hdfs, local FS, s3 etc.)
     UnderFileSystem ufs = UnderFileSystem.get(ufsAddress, tachyonConf);
 
+    if (!ufs.exists(ufsAddrRootPath.toString())) {
+      throw new FileNotFoundException("ufs path " + ufsAddrRootPath.toString() + " not found.");
```
Comment (calvinjia) [inline clients/unshaded/src/main/java/tachyon/client/UfsUtils.java]: You shouldn't need to do the `toString()`.

**Comment by calvinjia** [comment]
Jenkins, test this please.

**Comment by calvinjia** [comment]
ok to test

File: bin/tachyon-workers.sh (hunk 1 above)
Comment (kvaishnav) [inline bin/tachyon-workers.sh]: Yes, while starting tachyon .it just prints in a new line now. Cosmetic.

**Comment by kvaishnav** [comment]
Thanks Calvin. Removed toString() in exception.

File: clients/unshaded/src/main/java/tachyon/client/UfsUtils.java (hunk 2 above)
Comment (kvaishnav) [inline clients/unshaded/src/main/java/tachyon/client/UfsUtils.java]: Thanks.

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**